### DISCUSSION TITLE
Renovate app disabled for private GitHub repository

### DISCUSSION THREAD

**Discussion by qmg-mburns** [renovatebot/renovate#16887] [Request Help]
Hi,

I have a private repository on GitHub that has been showing as [disabled](https://app.renovatebot.com/dashboard#github/qmetric/qmg-quote). Would it be possible to re-enable this repo?
I think the issue might have been the related to our private maven repository - we've since fixed our side.

Thanks,

**Comment by nabeelsaabna** [comment]
hi @qmg-mburns, 
can you provide more details on how it got disabled ?

In general an org/repository admin should check this by:

1. go to https://github.com/settings/installations 
2. access the `Renovate` app configuration by pressing on `Configure` button
3. got to `Repository access` section 
4. update the list of selected repositories

**Reply by qmg-mburns** [reply]
hi @nabeelsaabna,

I'm not sure why the repository was disabled. I've checked the logs on the  [renovatebot dashboard](https://app.renovatebot.com/dashboard#github/qmetric/qmg-quote) and they seem to be empty.

I read somewhere that a potential reason for getting disabled is if the job runs for a long time? Potentially it could have been this as we didn't have our internal maven package manager accessible to Renovate at the time.

I've verified the `Repository access` configuration and the impacted repo is on the list of repositories.

**Reply by nabeelsaabna** [reply]
what is the renovate config you are using for this repo ?

**Reply by qmg-mburns** [reply]
```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base",
    ":disableHost(ci.qmetric.co.uk)"
  ],
  "packageRules": [
    {
      "matchManagers": ["maven"],
      "enabled": false
    }
  ]
}
```

**Reply by nabeelsaabna** [reply]
I'll need to check further and update on this

**Reply by rarkins** [reply]
I have removed the disabling for this repo but it will get disabled again if it continues to time out every job

**Reply by qmg-mburns** [reply]
All working now - thanks for your help.
